openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1408,10 +1337,7 @@ TCN_IMPLEMENT_CALL(void, SSL, setVerify)(TCN_STDARGS, jlong ssl, jint level, jin
     tcn_ssl_ctxt_t* c;
     SSL *ssl_ = J2P(ssl, SSL *);
 
-    if (ssl_ == NULL) {
-        tcn_ThrowException(e, "ssl is null");
-        return;
-    }
+    TCN_CHECK_NULL(ssl_, "ssl is null",);
```

Comment:
consider clarifying this as follow: `TCN_CHECK_NULL(ssl_, "ssl is null", /*void*/);`